app/schemas/models/user.coffee

```diff
@@ -111,6 +111,12 @@ _.extend UserSchema.properties,
         type: c.shortString() # E.g 'share progress modal parent'
         email: c.shortString()
         sent: c.date() # Set when sent
+
+    validations: c.array, { title: 'Sendgrid email validation results'},
```

Comment:
Want to confirm that this will not require the schema in any circumstances?  It's been so long since I've looked at a schema change, from what I remember and from comparing this to other fields this will not require it, but want to be sure.